<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>827</width>
    <height>352</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>0</y>
      <width>141</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item alignment="Qt::AlignVCenter">
      <widget class="QPushButton" name="AddEntry">
       <property name="text">
        <string>Add To Wallet</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="EditEntry">
       <property name="text">
        <string>Edit Entry</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="DeleteEntry">
       <property name="text">
        <string>Delete entry</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="LaunchEntry">
       <property name="text">
        <string>Launch Domain</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="Showpass">
       <property name="text">
        <string>Show Passwords</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>0</y>
      <width>621</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item alignment="Qt::AlignVCenter">
      <widget class="QPushButton" name="NewWallet">
       <property name="text">
        <string>Create New Wallet</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignVCenter">
      <widget class="QPushButton" name="CipherClose">
       <property name="text">
        <string>Cipher and Close</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignVCenter">
      <widget class="QPushButton" name="OpenCyphered">
       <property name="text">
        <string>Open Cyphered</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignVCenter">
      <widget class="QPushButton" name="EnvironmentBut">
       <property name="text">
        <string>Set Environment</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>621</width>
      <height>251</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QTableView" name="WalletView">
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignHCenter|Qt::AlignBottom">
      <widget class="QLabel" name="TableTitle">
       <property name="text">
        <string>Title</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLineEdit" name="UserFilter">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>210</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Filter by User</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="DomainFilter">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>240</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>125</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Filter by Domain</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="DescFilter">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>270</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Filter by Description</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="Months">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>170</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>-</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1 month</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2 months</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4 months</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>8 months</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1 year</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2 years</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Custom</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="CustomMonths">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>170</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Months</string>
    </property>
   </widget>
   <widget class="QLabel" name="olderText">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>150</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Passwords older than</string>
    </property>
   </widget>
   <zorder>layoutWidget</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>UserFilter</zorder>
   <zorder>DomainFilter</zorder>
   <zorder>verticalSpacer</zorder>
   <zorder>DescFilter</zorder>
   <zorder>Months</zorder>
   <zorder>CustomMonths</zorder>
   <zorder>olderText</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>827</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
